app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="chat.literacyapp.org.chat">
```

Comment:
I refactored the package name to "org.literacyapp.chat" in a previous commit: https://github.com/literacyapp-org/literacyapp-chat/commit/b45a0bca8254d0e8605c05f951b81cae986a8747